src/lib/Form/DataMapper/LanguageCreateMapper.php

```diff
@@ -8,17 +8,22 @@
 
 use eZ\Publish\API\Repository\Values\Content\LanguageCreateStruct;
 use eZ\Publish\API\Repository\Values\ValueObject;
+use EzSystems\EzPlatformAdminUi\Exception\InvalidArgumentException;
 use EzSystems\EzPlatformAdminUi\Form\Data\Language\LanguageCreateData;
 
 class LanguageCreateMapper implements DataMapperInterface
 {
     /**
-     * @param LanguageCreateStruct $value
-     *
+     * @param LanguageCreateStruct|ValueObject $value
      * @return LanguageCreateData
+     * @throws InvalidArgumentException
```

Comment:
Empty line between `@return` and `@throws`. I'd skip writting same comments below, just make sure you run `php-cs-fixer` on this commit.